server/course/course.controller.js

```diff
@@ -43,6 +43,7 @@ router
     try {
       const course = new Course(request.body);
       course.createdByUserID = request.user.userID;
+      course.reasonOfPlay = request.params.reasonOfPlay;
```

Comment:
why is reason being passed in as a param and not part of the json body in the request?

Reply:
Wooopsss